post-processor/README.md

```diff
@@ -0,0 +1,135 @@
+# IBM Image Exporter - Post-Processors
+
+## Scope
+The IBM Packer Post Processor Plugin can be used to Export Images on IBM Cloud.
+
+## Description
+The IBM Image Exporter post-processor will allow a customer to export to a COS bucket any VPC custom image they are authorized to that has become available and is not yet in the process of being deleted -- i.e. with status other than pending, failed, or deleting. 
+
+Generated **Image** from **Packer Builder** is exported to **IBM COS Bucker** if **image_id** is not specified in the post processor block.
+Also note if **image_id** is passed then **api_key** and **region** is mandatory.
+
+### Post-Processor
+- [ibmcloud-export](post-processor/ibmcloud-export) - The `ibmcloud-export` post-processor supports exporting custom images to COS bucket. 
+
+### Prerequisites
+Please refer to https://github.com/IBM/packer-plugin-ibmcloud/blob/5fe6132a8fd3d17e6e75c798a8ed7883496c25d5/README.md README.md file from the main repository section.
+
+## Usage
+
+Follow usage section of the README.md file till step 3 and follow the below step 4.
+
+4. Finally, run Packer plugin commands
+    ```shell
+    packer validate examples/postprocessor.vpc.rhel.pkr.hcl
+    packer build examples/postprocessor.vpc.rhel.pkr.hcl
+    ```
+
+***********
+
+## Packer Template in detail
+Packer's behavior is determined by the Packer template. This template tells Packer not only the plugins (builders, provisioners, post-processors) to use, but also how to configure them and in what order run them.
```

Comment:
Can we shuffle the order of this data structures of packer ?
like post processor>builder>provisioner ?

Reply:
No

Reply:
What ever order have specified in example folder